data/tlg0010/tlg023/tlg0010.tlg023.perseus-grc2.xml

```diff
@@ -4,8 +4,7 @@
 <teiHeader>
     <fileDesc>
         <titleStmt>
-            <title>To the Children of Jason</title>
-            
+            <title xml:lang="grc">τοῖς Ἰάσονος παισίν</title>            
```

Comment:
This time I took Greek titles right from the XML texts, which had non-capitalized titles for the letters.